server.py

```diff
@@ -0,0 +1,94 @@
+import requests
+# from flask import Flask
+from flask import Flask, request, render_template , send_file   
+import iitkgp_erp_login.erp as erp
+from utils import ERPSession
+from timetable import delete_calendar, create_calendar, build_courses, generate_ics
+import io
+
+from flask_cors import CORS
+
+app = Flask(__name__)
+CORS(app, support_credentials=True)
+session = requests.Session()
+headers = {
+   'timeout': '20',
+   'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/51.0.2704.79 Chrome/51.0.2704.79 Safari/537.36',
+}
+
+dict = {
+    'rollNo': 'sessionToken',
+}
+
+
+loginDetailsList = []
```

Comment:
instead of having `loginDetailsList` list, try making a separate class to manage sessions in a separate file